.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,93 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: 🐞 Bug report
+about: Report a bug
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+/kind bug
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+<!--
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+Welcome! - We kindly ask you to:
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+  1. Fill out the issue template below 
+  2. Use Gitter if you have a question rather than a bug or feature request.
+    Gitter community channel is at: https://gitter.im/devfile/community
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+Thanks for understanding, and for contributing to the project!
```

Comment:
Again, NBD, so no need to change, but the comma here is unnecessary. It can simply be, `Thanks for understanding and for contributing to the project!`

Reply:
updated